src/cephadm/cephadm

```diff
@@ -4941,7 +4942,9 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--apply-spec',
         help='Apply cluster spec after bootstrap (copy ssh key, add hosts and apply services)')
-
+    parser_bootstrap.add_argument(
```

Comment:
This option is something that should be included inside the cluster spec file, not as parameter of a more general command ( bootstrap).
Take into account that this is only valid when you want to apply a cluster specification.
We are trying to reduce the number of options in the cephadm commands, this not help at all. So another reason to kindly suggest to you to move it inside the ceph cluster spec file

Note: Options with a negation in the name shouldn't be easy to use.

Reply:
That makes perfect sense, thanks - I'll update it soon.

Reply:
Ok, I'll need some help - where should the option be added? I guess I should update the _add_host function in src/pybind/mgr/cephadm/module.py - add the ssh-copy-id there, based on a label (like ssh-copy-key)? And remove the ssh-copy-id in src/cephadm/cephadm? Or is there a better way?